docs/_pages/docs/03-understand/03-premium.md

```diff
@@ -8,4 +8,46 @@ sidebar:
 src: "_pages/docs/03-understand/03-premium.md"
 ---
 
-{% include wip %}
\ No newline at end of file
+# **Summary**
+
+Order makers choose any premium (positive/negative/zero) when buying or selling bitcoin. This premium can be either "relative" to the current bitcoin-fiat market rate or an "explicit" amount using a fixed number of satoshis.
```

Comment:
I would start by defining in simple terms what is the premium: e.g., the cost over/under the regular CEX BTC price for that p2p order.

Technically, there is two pricing methods: relative and explicit. The premium is only specified when the "relative" pricing method is selected. The fixed amount of Satoshis is given when the pricing is "explicit".

Reply:
I agree, such a simple thing I overlooked. I've revised the summary to first define what is a premium.

Thank you for clearing up that relative/explicit pricing method. I misunderstood when I wrote this but I've revised it to reflect your comments.